CMakeLists.txt

```diff
@@ -30,7 +30,10 @@ if (NOT DEFINED LF_REACTOR_CPP_SUFFIX)
   endif()
 endif()
 
-find_package (Threads)
+find_package(Threads)
+
+find_package(Backtrace)
```

Comment:
Why not hinding it behind some build flag?

Reply:
I assume you mean 'hiding', but I don't understand what you would like to hide? This PR effectively enables backtracing support automatically if an implementation of `backtrace()` is found.